pkg/cri/server/sandbox_stats_list.go

```diff
@@ -18,8 +18,11 @@ package server
 
 import (
 	"context"
+	"errors"
 	"fmt"
 
+	"github.com/containerd/ttrpc"
```

Comment:
nit: import ordering. "github.com/containerd/ttrpc" should be part of the below group

Reply:
Done!